tests/unit/test_cli.py

```diff
@@ -905,24 +943,40 @@ def test_a_user_can_successfully_merge_several_cachi2_sboms(
         invoke_expecting_sucess(app, ["merge-sboms", *sbom_files_to_merge])
 
     @pytest.mark.parametrize(
-        "sbom_files_to_merge",
+        "sbom_files_to_merge,extra_options",
         [
-            [
-                "./tests/unit/data/sboms/cachi2.bom.json",
-                "./tests/unit/data/sboms/cachito_gomod.bom.json",
-            ],
-            [
-                "./tests/unit/data/sboms/cachi2.bom.json",
-                "./tests/unit/data/sboms/cachito_gomod.bom.json",
-                "./tests/unit/data/sboms/cachito_gomod_nodeps.bom.json",
-            ],
+            (
+                [
+                    "./tests/unit/data/sboms/cachi2.bom.json",
+                    "./tests/unit/data/sboms/cachito_gomod.bom.json",
+                ],
+                [],
+            ),
+            (
+                [
+                    "./tests/unit/data/sboms/cachi2.bom.json",
+                    "./tests/unit/data/sboms/cachito_gomod.bom.json",
+                    "./tests/unit/data/sboms/cachito_gomod_nodeps.bom.json",
+                ],
+                [],
+            ),
+            (
+                [
+                    "./tests/unit/data/sboms/cachi2.bom.spdx.json",
+                    "./tests/unit/data/sboms/syft.bom.spdx.json",
+                ],
+                ["--sbom-type", "spdx"],
+            ),
         ],
     )
     def test_a_user_can_successfully_save_sboms_merge_results_to_a_file(
         self,
         sbom_files_to_merge: list[str],
+        extra_options: list[str],
```

Comment:
Same request as on L625 here. Also I suggest renaming `extra_options` to `sbom_type_option` or something that clearer states the intended use.